### ISSUE TITLE
FCN8

### ISSUE THREAD

**Issue by GYE19970220** [yassouali/pytorch_segmentation#61]
In the forward of fcn.py,why pool4 need to multiply 0.01,pool3 need to multiply 0.0001,like
`adjstd_pool4 = self.adj_pool4(0.01 * pool4)`
`adjstd_pool3 = self.adj_pool3(0.0001 * pool3)`

**Comment by yassouali** [comment]
Hi, if I was remember correctly, it was used to have more stable gradients, but I do think not using it will not change things significantly, maybe I'll use vgg16 with batch norm and delete this. Thanks for the reminder.
